server/config/subject_page.proto

```diff
@@ -31,9 +43,11 @@ message PageMetadata {
   // Contained places to load for each parent place type.
   // This can be overwritten by tiles.
   map<string, string> contained_place_types = 4;
+  // Event type information to be used by this config. The key is the event
+  // type id and the value is the spec for that event type.
+  map<string, EventTypeSpec> event_type_spec = 5;
```

Comment:
this is interesting. we should consider moving stat_var_spec here later, since it could sometimes be used across categories.